### DISCUSSION TITLE
Getting Started Tutorial?

### DISCUSSION THREAD

**Discussion by guw** [eclipse-oomph/oomph#23] [Q&A]
I'm looking for some guidance on how to best get started with Oomph. My goal is to write a setup thing that creates a full Eclipse installation (with some plug-ins) and a workspace. The workspace should have a project populated and some preferences applied. That setup thing should be checked into source control so other team members can use it. 

I started looking at the documentation: [Oomph Authoring](https://wiki.eclipse.org/Eclipse_Oomph_Authoring). It explains the dialogs and a lot things. But is there a simpler tutorial? I am also a bit confused with the many options in the Oomph category. There is much more than what I see in the documentation or on old tutorials.

<img width="586" alt="image" src="https://github.com/eclipse-oomph/oomph/assets/191472/f0bd9879-3b41-481f-ae76-d2c3a676f16a">

Which model so I start with these days? Still project model or something else?

**Comment by dancer1325** [comment]
After spending 1 hour to identify OFFICIAL documentation, rather than the only https://wiki.eclipse.org/Oomph, I found https://download.eclipse.org/oomph/help/, which at the end it's hosted in https://github.com/eclipse-oomph/oomph

**Reply by merks** [reply]
It's a bit embarrassing that there is an unanswered question and that I forgot about this documentation.  It's probably not so well maintained.  😞
